packages/core/src/types/IViewport.ts

```diff
@@ -7,6 +7,20 @@ import ViewportType from '../enums/ViewportType';
 import ViewportStatus from '../enums/ViewportStatus';
 import DisplayArea from './displayArea';
 
+/**
+ * Specifies what image to get a reference for.
+ */
+export type TargetSpecifier = {
+  /** The slice index within the current viewport camera to get a reference for */
+  sliceIndex?: number;
+  /** True to get a frame of reference UID reference instead of a regular image one */
+  forFrameOfReference?: boolean;
+  /** Set of points to get a reference for, in world space */
+  points?: Point3[];
```

Comment:
I'm not sure what this `points` is really. "set of points to get a reference for" is not really clear to me. Can you remove it until you show how you use it?